src/Message.php

```diff
@@ -20,155 +25,227 @@
+        $new->headerColor = $value;
         return $new;
     }
 
     /**
-     * Returns a new instance with the specified options for rendering the close button tag.
+     * The header message in the message component. Message widget will also be treated as the header content, and will
+     * be rendered before body.
      *
-     * @param array $value The close button is displayed in the header of the modal window. Clicking on the button
-     * will hide the modal window. If {@see withoutCloseButton} is false, no close button will be rendered.
+     * @param string $value The header message.
      *
      * @return self
      */
-    public function closeButtonOptions(array $value): self
+    public function headerMessage(string $value): self
     {
         $new = clone $this;
-        $new->closeButtonOptions = $value;
+        $new->headerMessage = $value;
         return $new;
     }
 
     /**
-     * Returns a new instance with the specified HTML attributes for the widget header tag.
+     * The HTML attributes for the widget header tag.
+     *
+     * @param array $values Attribute values indexed by attribute names.
      *
-     * @param array $value The HTML attributes for the widget header tag.
+     * @return self
      *
      * {@see Html::renderTagAttributes()} for details on how attributes are being rendered.
+     */
+    public function headerAttributes(array $values): self
+    {
+        $new = clone $this;
+        $new->headerAttributes = $values;
+        return $new;
+    }
+
+    /**
+     * Returns a new instance with the specified ID of the widget.
+     *
+     * @param string $value The ID of the widget.
      *
      * @return self
      */
-    public function headerOptions(array $value): self
+    public function id(string $value): self
     {
         $new = clone $this;
-        $new->headerOptions = $value;
+        $new->attributes['id'] = $value;
         return $new;
     }
 
     /**
-     * Returns a new instance with the specified size message widget.
+     * Set size config widgets.
      *
-     * @param string $value The size message widget. Default setting empty normal, 'is-small', 'is-medium', 'is-large'.
+     * @param string $value size class.
      *
      * @return self
+     *
+     * @link https://bulma.io/documentation/components/message/#sizes
      */
     public function size(string $value): self
     {
+        if (!in_array($value, self::SIZES)) {
+            $values = implode(' ', self::SIZES);
+            throw new InvalidArgumentException("Invalid size. Valid values are: $values.");
+        }
+
         $new = clone $this;
         $new->size = $value;
         return $new;
     }
 
     /**
-     * Returns a new instance with the disabled close button message widget.
+     * Allows you to disable close button message widget.
      *
      * @return self
      */
-    public function closeButton(): self
+    public function unclosedButton(): self
```

Comment:
```suggestion
    public function withoutCloseButton(): self
```

Also, majority of options now accept boolean value.

Reply:
done.